hack/monitor_prow.py

```diff
@@ -67,11 +67,14 @@ def highlight(log_dir, dc):
             if pod.startswith('deck-internal'):
                 cmd += ['--container', 'deck']
             debug("deployment/{}: pod/{}: getting logs".format(dc, pod))
-            for l in exec_cmd(*cmd).splitlines():
-                if warn in l:
-                    log_lines.append(YELLOW + l + RESET)
-                elif error in l or fatal in l:
-                    log_lines.append(RED + l + RESET)
+            try:
+                for l in exec_cmd(*cmd).splitlines():
+                    if warn in l:
+                        log_lines.append(YELLOW + l + RESET)
+                    elif error in l or fatal in l:
+                        log_lines.append(RED + l + RESET)
+            except:
```

Comment:
```suggestion
            except subprocess.CalledProcessError:
```

so we don't catch other potential errors.

https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError

Reply:
Oh, pylint won.

Reply:
pylint is da best!